src/foam/u2/view/ReadReferenceView.js

```diff
@@ -28,17 +92,41 @@ foam.CLASS({
       expression: function(obj) { return obj.cls_; }
     },
     'prop',
+    {
+      class: 'String',
+      name: 'linkTo',
+      documentation: 'link to the reference'
```

Comment:
Can you expand upon or improve this documentation. I don't find the explanation clear.